### PR TITLE
Refactor/part 19.UI ux enhancement

### PR DESCRIPTION
**Pull request by phucvarmeta**

### CODE REVIEW

File: packages/web/src/common/utils/date-util.ts [hunk 1]
```diff
@@ -6,6 +6,7 @@ const MIN_TIME = 60 * 1000;
 const SEC_TIME = 1000;
 
 export const getDateUtcToLocal = (d: any) => {
+  console.log("🚀 ~ getDateUtcToLocal ~ d:", d);
```
Comment (AkydA) [inline packages/web/src/common/utils/date-util.ts:9]: please remove `console.log`

File: packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx [hunk 2]
```diff
@@ -685,6 +685,13 @@ const EarnAddLiquidityContainer: React.FC = () => {
     return isFetchingPools || isLoadingCommon;
   }, [isFetchingPools, isLoadingCommon]);
 
+  const hasStakingRewardPool = useMemo(
+    () =>
+      selectPool.poolPath ===
+      "gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000",
```
Comment (AkydA) [inline packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx:691]: Is `gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000` only pool which has staking reward?

File: packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx [hunk 3]
```diff
@@ -829,6 +829,13 @@ const EarnAddLiquidityContainer: React.FC = () => {
     return isFetchingPools || isLoadingCommon;
   }, [isFetchingPools, isLoadingCommon]);
 
+  const hasStakingRewardPool = useMemo(
+    () =>
+      selectPool.poolPath ===
+      "gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000",
```
Comment (AkydA) [inline packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx:835]: Is `gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000` only pool which has staking reward?

File: packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx (hunk 2 above)
Comment (phucvarmeta) [reply packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx:691]: https://varmeta.slack.com/archives/C06MXPW7T0A/p1721619673555989?thread_ts=1721617118.020319&cid=C06MXPW7T0A

File: packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx (hunk 3 above)
Comment (phucvarmeta) [reply packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx:835]: https://varmeta.slack.com/archives/C06MXPW7T0A/p1721619673555989?thread_ts=1721617118.020319&cid=C06MXPW7T0A

Comment (phucvarmeta) [reply packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx:835]: i think i should change to another name

File: packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx (hunk 2 above)
Comment (phucvarmeta) [reply packages/web/src/containers/pool-add-liquidity-container/PoolAddLiquidityContainer.tsx:691]: i think i should change to another name

File: packages/web/src/hooks/reposition/use-reposition-handle.tsx [hunk 4]
```diff
@@ -363,6 +363,7 @@ export const useRepositionHandle = () => {
     }
     return Number(amountB) - repositionAmountB;
   }, [currentAmounts, repositionAmounts]);
+  console.log("🚀 ~ swapAmount ~ swapAmount:", swapAmount);
```
Comment (AkydA) [inline packages/web/src/hooks/reposition/use-reposition-handle.tsx:366]: here is another console log

File: packages/web/src/hooks/reposition/use-reposition-handle.tsx [hunk 5]
```diff
@@ -388,13 +389,19 @@ export const useRepositionHandle = () => {
       exactType: "EXACT_IN" as const,
     };
   }, [currentAmounts, repositionAmounts, selectedPosition, swapAmount]);
+  console.log(
```
Comment (AkydA) [inline packages/web/src/hooks/reposition/use-reposition-handle.tsx:392]: here is another console log

File: packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx [hunk 6]
```diff
@@ -829,6 +829,13 @@ const EarnAddLiquidityContainer: React.FC = () => {
     return isFetchingPools || isLoadingCommon;
   }, [isFetchingPools, isLoadingCommon]);
 
+  const showOneClickStaking = useMemo(
+    () =>
+      selectPool.poolPath ===
+      "gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000",
```
Comment (AkydA) [inline packages/web/src/containers/earn-add-liquidity-container/EarnAddLiquidityContainer.tsx:835]: I mean `poolPath` is hardcoded here. and this code only affect to `gno.land/r/demo/wugnot:gno.land/r/gnoswap/gns:3000`

Report to Jaeho if you don't have way to get data about whether pool is stakable

**Review by AkydA — APPROVED** [review]

**Pull request merged**